.github/workflows/on_add_issue.yml

```diff
@@ -0,0 +1,30 @@
+name: Add new issues to the Delivery Assurance project board
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  initialize:
+    name: Initialize new issue
+    runs-on: ubuntu-latest
+    steps:
+      # Rather than use a personal access token to interact with the project, we
+      # can use this GitHub App. There's an API for exchanging app credentials
+      # for a short-term token, and we use that API here.
```

Comment:
For my own understanding -- this step generates a token that grants permissions for the DA project board repo, not for the eng-hiring site repo.

Reply:
It actually grants permissions for a GitHub app that has been created for the org

@mgwalker set it up, so he'll have more details, but here's the [relevant slack thread](https://gsa-tts.slack.com/archives/C02CD5VUQ/p1672154090789699) and the [blog post referenced](https://josh-ops.com/posts/github-apps/ ) that explains GitHub apps in more detail

Reply:
@Sgtpluck Thank you! Might be helpful to add some notes to that effect in the code comments?